spec/rhc/commands/snapshot_spec.rb

```diff
@@ -32,14 +32,26 @@
 
     context 'when saving a snapshot' do
       before do
+        RHC::Helpers.should_receive(:agree).with(/Do you want to overwrite this file?/).and_return(true)
```

Comment:
The test apparently creates the file in advance so the file already exists:
`FileUtils.cp(File.expand_path('../../assets/targz_sample.tar.gz', __FILE__), @targz_filename)`